pages.de/common/git-commit.md

```diff
@@ -7,6 +7,10 @@
 
 `git commit -m "{{nachricht}}"`
 
+- Committe gestagten Dateien zum Repository mit einer Nachricht aus einer Datei:
```

Comment:
```suggestion
- Committe gestagte Dateien zum Repository mit einer Nachricht aus einer Datei:
```
```suggestion
- Committe alle gestagten Dateien zum Repository mit einer Nachricht aus einer Datei:
```

Not sure which one would be correct/better here.

Reply:
I like the second one better